shared/message/support/FieldDescription.proto

```diff
@@ -22,42 +22,83 @@ package message.support;
 
 import "Vector.proto";
 
+/**
+ * Information about location and size of field objects and markers.
+ * Details about field measurements can be found in 'Law 1 - The Field of Play',
+ * in the RoboCup rulebook.
+ * We have these config values mirrored in `SoccerConfig/data/config/`.
+ */
 message FieldDescription {
+    /// The cross-sectional shape of the goalposts
     enum GoalpostType {
         RECTANGLE = 0;
         CIRCLE    = 1;
     }
-
+    /// Measurements of the field which are loaded from config file. In metres
     message FieldDimensions {
-        double       line_width             = 1;
-        double       mark_width             = 2;
-        double       field_length           = 3;
-        double       field_width            = 4;
-        GoalpostType goalpost_type          = 5;
-        double       goal_depth             = 6;
-        double       goal_width             = 7;
-        double       goal_area_length       = 8;
-        double       goal_area_width        = 9;
-        double       goal_crossbar_height   = 10;
-        double       goalpost_width         = 11;
-        double       goalpost_depth         = 12;
-        double       goal_crossbar_width    = 13;
-        double       goal_crossbar_depth    = 14;
-        double       goal_net_height        = 15;
-        double       penalty_mark_distance  = 16;
-        double       center_circle_diameter = 17;
-        double       border_strip_min_width = 18;
-    }
+        /// The rules say that the lines must be the same width - approximately
+        /// 5 cm, and no more than 12 cm
+        double line_width = 1;
+
+        double mark_width = 2;
```

Comment:
I assume the penalty mark?

Reply:
It used to be specified in the rules, back when that field was added. Now they don't specify the mark size in the rules - I presume it's implied to be the same size as the lines. I'll remove this field (we don't use it anyway)